cs/test/core/Util.cs

```diff
@@ -271,6 +273,22 @@ public static ArraySegment<byte> SerializeUnsafeCB<T>(T obj)
             return output.Data;
         }
 
+        public static ArraySegment<byte> SerializeUnsafeCBWithPtr<T>(T obj)
```

Comment:
```
Managed Serialize -> Unmanaged deserialize
Unmanaged Serialize -> Unmanaged deserialize
```
Does not imply:
```
Unmanaged Serialize -> Managed Deserialize
```

It is OK for me, if you are convinced enough there is no posibility it won't introduce problems down the line. Just wanted to note that would be required for completeness.